auth.go

```diff
@@ -204,12 +205,11 @@ func (n krpAuthorizerAttributesGetter) GetRequestAttributes(u user.Info, r *http
 	return attrs
 }
 
-type Handler struct {
-	AuthInterface
-	Config AuthConfig
+type Handler interface {
```

Comment:
This would be an `AuthHandler` as it returns whether authN and authZ was successful.